tests/test_e2e_07_topology.py

```diff
@@ -0,0 +1,291 @@
+import asyncio
+import datetime
+import httpx
+import time
+from datetime import datetime, timedelta, UTC, timezone
+
+
+from .helpers import NetworkTest
+
+CONTROLLER = '127.0.0.1'
+KYTOS_API = 'http://%s:8181/api' % CONTROLLER
+
+TIME_FMT = "%Y-%m-%dT%H:%M:%S+0000"
+
+class TestE2EMefEline:
+    net = None
+
+    @classmethod
+    def setup_class(cls):
+        cls.net = NetworkTest(CONTROLLER)
+        cls.net.start_controller(clean_config=True, enable_all=False)
+        time.sleep(5)
+
+    @classmethod
+    def teardown_class(cls):
+        cls.net.stop()
+
+    def setup_method(self, method):
+        self.net.start_controller(clean_config=True, enable_all=False)
+        time.sleep(5)
+
+    def teardown_method(self, method):
+        pass
+
+    async def test_010_topology_race_condition(self):
```

Comment:
Can we have also a more descriptive test case name reflecting which race cond is this supposed to cover? 

The test case itself is also a bit lower-ish level since it's inserting directly in the DB, but certainly acceptable for what we want to have coverage for, so let's also add docstring explaining the overall test case goal.